### DISCUSSION TITLE
Help! The home screen crazy crash. Can not uninstall, show File system error (-1073741571)

### DISCUSSION THREAD

**Discussion by ZYV5ge** [valinet/ExplorerPatcher#2526] [Q&A]
Try to uninstall, after the new window preview (build 26002), the home screen crash go crazy. Try to uninstall it. 
Every way to uninstall not working. 
show File system error (-1073741571)

**Comment by pyrates999** [comment]
Canary builds are not supported because the code is not final and can change with each new release.

**Reply by ZYV5ge** [reply]
Can you give me advice to manual renove the software? Thanks

**Comment by pyrates999** [comment]
To uninstall, you can do one of the following:

- Use "Programs and Features" in Control Panel, or "Apps and features" in the Settings app or
- Run ep_setup.exe /uninstall or
- Rename ep_setup.exe to ep_uninstall.exe and run that.
